services/movie.py

```diff
@@ -22,11 +26,12 @@
     return Movie.objects.get(id=movie_id)
 
 
+@transaction.atomic
 def create_movie(
-    movie_title: str,
-    movie_description: str,
-    genres_ids: list = None,
-    actors_ids: list = None,
+        movie_title: str,
+        movie_description: str,
+        genres_ids: list = None,
+        actors_ids: list = None,
```

Comment:
The type hints for `genres_ids` and `actors_ids` should be more specific. Consider using `list[int]` to match the type hints used in the `get_movies` function.